docker-compose.yaml.template

```diff
@@ -19,6 +20,15 @@ services:
     deploy:
       labels:
         - "prometheus-job=save-backend"
+  preprocessor:
+    image: 127.0.0.1:6000/save-preprocessor:{{project.version}}
+    volumes:
```

Comment:
I believe you should have a separate sections for volumes, as describe in [this section of docs](https://docs.docker.com/storage/volumes/#use-a-volume-with-docker-compose). And that volume can be mounted only to required containers and not exposed to host system.